### ISSUE TITLE
Display some order info on the "create pick" form

### ISSUE THREAD

**Issue by alexgleason** [HarvestHub/GardenHub#83]
I think it would be helpful to the picker if in addition to what they see (pictured below) they also see some important information about the order on this page. Maybe the list of crops, the person who requested the order, etc.

That way if they need that information they don't have to back out of the "create pick" view to see it.

![screen shot 2018-01-11 at 18 26 53](https://user-images.githubusercontent.com/3639540/34852545-3b434e8c-f6fd-11e7-9083-7eef40d1838c.png)

**Comment by marykatefain** [comment]
Agree

**Comment by marykatefain** [comment]
Additionally, would be great if they only saw the crops they are *supposed* to be picking in the drop down:

![screenshot from 2018-01-11 18 59 22](https://user-images.githubusercontent.com/17955536/34853426-a13b28be-f701-11e7-82d1-5db7302fbe72.png)

**Comment by alexgleason** [comment]
> Additionally, would be great if they only saw the crops they are supposed to be picking in the drop down

Well, submitting an Order and submitting a Pick are pretty different. For instance, let's say we implement a "pick all crops" feature on the database side that actually stores that as a database column for an order. At the same time, the Plot on the Order might not have an updated list of crops. So you could submit an order where it's actually picked differently than what the "crops" field in the order form (or the garden) says.

Another example of this would be if the requester wrote something extra in the comments section, or called GardenHub asking them to do something differently, or maybe Jeff the picker just knows that Alan the gardener always wants his tomatoes picked.

Basically, I think a Pick shouldn't be constrained by anything except that it's a record of a list of crops picked from a certain plot at a certain time. Otherwise we're going to have some really annoyed pickers.

**Comment by marykatefain** [comment]
Good points.  :+1:

**Comment by alexgleason** [comment]
Here's what I've got. Bonus: you can tap the little crop icons and it automatically adds them to the list. Added in 22849927a644d19a6358e6b9ee256ca7d0e4bc2d.

![127 0 0 1_8000_pick_3_ nexus 5 1](https://user-images.githubusercontent.com/3639540/35190819-09cf4dfe-fe39-11e7-85a6-817365f91a90.png)

**Comment by marykatefain** [comment]
Love this!!! Good idea.

**Comment by JenniferGold** [comment]
Nice!!!
